src/App.js

```diff
@@ -1,5 +1,7 @@
 import "./App.css";
 import { Box, Image, Badge, Flex, Spacer } from "@chakra-ui/react";
+import {StarIcon,ChatIcon,ExternalLinkIcon,DownloadIcon} from '@chakra-ui/icons'
```

Comment:
Hey great libraty, I think we should install a linter and a hook for fixing the code style automatically before pushing. That way, we could have an standard format for spaces around curly brackets `{foo}` vs `{ foo }`